persistent-template/Database/Persist/TH.hs

```diff
@@ -1116,6 +1082,7 @@ mkEntity entMap mps t = do
 
 mkUniqueKeyInstances :: MkPersistSettings -> EntityDef -> Q [Dec]
 mkUniqueKeyInstances mps t = do
+    -- FIXME: isExtEnabled breaks the benchmark
```

Comment:
It should be possible to just add the flag to the benchmark.

Benchmarks should be built in CI - oops!

Reply:
I take it this doesn't require any action on my part?

Reply:
Correct :)